source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -60,10 +76,10 @@ by adding the bucket region parameter to the S3 URL:
 .. code-block:: none
 
   plugin {
-    obox_index_fs = compress:gz:6:s3: https://ACCESSKEY:SECRET@host/?bucket=BUCKETNAME&region=eu-central-1
+    obox_index_fs = compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@host/?bucket=BUCKETNAME&region=eu-central-1
   }
 
-The s3 driver also supports the bulk-deletion. The bulk-delete is enabled by default to delete up to 1000 keys with one request. To change this behaviour refer to ``bulk_delete_limit`` at :ref:`http_based_object_storages`
+The ``aws-s3`` and ``s3`` driver also support bulk-deletion. The bulk-delete is enabled by default to delete up to 1000 keys with one request. To change this behaviour refer to ``bulk_delete_limit`` at :ref:`http_based_object_storages`. Bulk delete can only efficiently run on multiple objects if you allow it to, via setting ``obox_max_parallel_deletes`` greater one (refer to :ref:`obox_settings`).
```

Comment:
s/driver also support/drivers support/
s/bulk-delete/``bulk-delete`` option/
s/you allow it to/configured to do so/

Reply:
thanks